packages/strapi-tiptap-editor/admin/src/components/Wysiwyg/index.tsx

```diff
@@ -208,7 +208,23 @@ const WysiwygContent = ({
       : null,
     settings.other && settings.other.wordcount ? CharacterCountExtension.configure() : null,
     settings.youtube.enabled
-      ? YouTubeExtension.configure({
+      ? YouTubeExtension.extend({
+          addAttributes() {
+            return {
+              ...this.parent?.(),
+              'data-title': {
+                default: null,
+                parseHTML: (element) => element.getAttribute('data-title'),
```

Comment:
Hoeft niet per se nu aangepast te worden, maar the nice thing about `data-` attributes is dat je ze kan opvragen via [element.dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset)